tests/worker_coordinator/runner_test.py

```diff
@@ -1154,122 +1154,94 @@ async def test_force_merge_with_params(self, opensearch, on_client_request_start
 
         opensearch.indices.forcemerge.assert_called_once_with(index="_all", max_num_segments=1, request_timeout=50000)
 
-    @mock.patch('osbenchmark.client.RequestContextHolder.on_client_request_end')
-    @mock.patch('osbenchmark.client.RequestContextHolder.on_client_request_start')
-    @mock.patch("opensearchpy.OpenSearch")
-    @run_async
-    async def test_force_merge_with_polling_no_timeout(self, opensearch, on_client_request_start, on_client_request_end):
-        opensearch.indices.forcemerge.return_value = as_future()
-
-        force_merge = runner.ForceMerge()
-        await force_merge(opensearch, params={"index" : "_all", "mode": "polling", 'poll-period': 0})
-        opensearch.indices.forcemerge.assert_called_once_with(index="_all")
-
```

Comment:
Should there be a test where a connection timeout is thrown when a long poll period is in effect?

Reply:
This will be applicable to every api. i don't see connection timeout was specifically called out. In previous case, i understand it could be a problem due to synchronous task. I don't think it is problem in this case. However, if you have strong feeling i can write another test that raise connection timeout error.

Reply:
Not a major issue; I will leave it up to you.  It was just a suggestion.